frontend/src/api/api-requests.jsx

```diff
@@ -0,0 +1,46 @@
+import config from '../../config.json';
+
+//TODO: check userID, generate tripID
+
+export const postTrip = async (trip) => {
+    console.log(JSON.stringify({"trip": trip}))
+    const response = await fetch(config.baseURL + "trips/1", {
+        method: "POST",
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({"trip": trip})
+    });
+    const data = await response.json();
+    console.log(data);
+}
+
+export const getAddrDetails = async (address) => {
+    const response = await fetch(
+        config.baseURL + "place?" + new URLSearchParams({
+            "placeID": address
+        }), 
+        {
+            method: "GET"
+        }
+    );
+    const data = await response.json();
+    console.log(data);
+    return data;
+}
+
+export const optimizeRoute = async (trip) => {
+    if (trip.length < 3) {
```

Comment:
Lol why?

Reply:
well im setting the first point in the list as the "starting point", so having either 1 or 2 points is trivial because the first point is always gonna stay the same lol. gonna make it so u can select the starting point